pymc3/examples/GHME_2013.py

```diff
@@ -1,20 +1,10 @@
-# -*- coding: utf-8 -*-
-# <nbformat>3.0</nbformat>
-
-# <codecell>
+import numpy as np
 import pandas as pd
-from pylab import *
+import pylab as pl
```

Comment:
I thought the convention is `import matplotlib.pyplot as plt`? I don't really care, other than for consistency.